### PR TITLE
fix #2364 - fix FT.SEARCH RETURN []

### LINKED ISSUE #2364: Node Redis cannot return empty documents
When executing the following code:

```javascript
const results = await nodeRedis.ft.search('test:index', '*', { RETURN: [] })
```

I get the following error:

```
/Users/guyroyse/code/playground/ioredis-tests/node_modules/@redis/search/dist/commands/SEARCH.js:18
            value: tuples.length === 2 && tuples[0] === '$' ?
                          ^

TypeError: Cannot read properties of undefined (reading 'length')
    at Object.transformReply (/Users/guyroyse/code/playground/ioredis-tests/node_modules/@redis/search/dist/commands/SEARCH.js:18:27)
    at transformCommandReply (/Users/guyroyse/code/playground/ioredis-tests/node_modules/@redis/client/dist/lib/commander.js:88:20)
    at Commander.commandsExecutor (/Users/guyroyse/code/playground/ioredis-tests/node_modules/@redis/client/dist/lib/client/index.js:177:54)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async rediSearchResult (file:///Users/guyroyse/code/playground/ioredis-tests/server.js:66:19)
    at async file:///Users/guyroyse/code/playground/ioredis-tests/server.js:71:1
```

It looks like Node Redis doesn't handle an empty RETURN.

**Environment:**
 - **Node.js Version**: 16.16.0
 - **Redis Server Version**: 6.2.7
 - **Node Redis Version**: 4.5.1
 - **Platform**: Mac OS 12.6

### PR DESCRIPTION
**Pull request by leibale**
fix #2364 - fix FT.SEARCH RETURN []

### CODE REVIEW

File: packages/search/lib/commands/SEARCH.ts [hunk 1]
```diff
@@ -72,14 +71,13 @@ export function transformArguments(
 export type SearchRawReply = Array<any>;
 
 export function transformReply(reply: SearchRawReply): SearchReply {
+    console.log(reply);
```
Comment (guyroyse) [inline packages/search/lib/commands/SEARCH.ts:74]: Probably want to remove this.

**Review by guyroyse — CHANGES_REQUESTED** [review]
LGTM except for the stray console.log.

**Review by guyroyse — APPROVED** [review]

**Review by guyroyse — APPROVED** [review]
LGTM

**Comment by TheArmagan** [comment]
Please approve that pull request its needed.

**Review by TheArmagan — APPROVED** [review]
everything looks ok to me

**Comment by leibale** [comment]
This will get released once #2373  is ready

**Pull request merged**